### PR TITLE
Add canonicalOrigin support to the CID API.

### PR DESCRIPTION
**Pull request by lannka**

### CODE REVIEW

File: src/service/cid-api.js [hunk 1]
```diff
@@ -123,6 +129,7 @@ export class GoogleCidApi {
   fetchCid_(url, scope, token) {
     const payload = dict({
       'originScope': scope,
+      'canonicalOrigin': this.canonicalOrigin_,
```
Comment (lannka) [inline src/service/cid-api.js]: @Gregable theoretically canonicalOrigin can be `null` if there is misconfiguration (might only happen when the page is invalid AMP). what does CID API server do in that case?

Comment (Gregable) [reply src/service/cid-api.js]: It should ignore the field and use the Host header instead.

**Comment by lannka** [comment]
Don't merge now. Wait for server side change being deployed first.

**Comment by lannka** [comment]
Actually server is already accepting the new field.

/cc @Gregable

**Pull request merged**